modules/doc/content/application_usage/tracked_apps.md

```diff
@@ -1,44 +1,136 @@
-- [Moltres](https://github.com/arfc/moltres) - Molten salt reactor simulation
+
+
+### Chemistry & species transport
+
 - [DGOSPREY](https://github.com/aladshaw3/dgosprey) - Discontinuous Galerkin Off-gas SeParation and REcoverY model: joint development between INL and GIT
+- [MAMBA](https://github.com/shortlab/mamba) - CASL's code for boiling, heat transfer, and chemistry in porous media
+- [TMAP8](https://mooseframework.inl.gov/TMAP8/) - System-level mass and thermal transport calculations related to tritium migration.
+
+
+### Geomechanics, Ground Flow, and Earth Sciences
+
+- [FalCon](https://github.com/idaholab/falcon) - Fracturing and liquid conservation; geothermal reservoir simulation and analysis code for coupled and fully implicit Thermo-Hydro-Mechanical-Chemical (THMC) geosystems
+- [Redback](https://github.com/pou036/redback) - Rock Mechanics with Dissipative (Thermo-Hydro-Mechanical-Chemical) Feedbacks: joint development between CSIRO and UNSW Australia
+
+
+### Miscellaneous
+
+- [moopy](https://github.com/aurora-multiphysics/moopy) - Python interface that simplifies (ish) the creation of MOOSE inputs
+- [Proteus](https://github.com/aurora-multiphysics/proteus) - For developing Fusion Digital Twins.
+- [Ranger](https://github.com/idaholab/ranger) - Python-based auto-response bot that uses the GitHub API and LlamaIndex package to monitor and generate relevant responses for new discussions
+- [Virtual Test Bed](https://mooseframework.inl.gov/virtual_test_bed/) - An open repository of simulations of nuclear systems
+- [Isopod](https://github.com/idaholab/isopod) - Multiphysics PDE constrained optimization, mostly merged into MOOSE as the optimization module
+
+
+## Closed Source Applications
+
+These applications require a license. Many of them can be obtained through the [NCRC website](https://ncrcaims.inl.gov/).
+
+### Nuclear Engineering Advanced Modeling and Simulation (NEAMS) tools
+
+- [BISON](https://inlsoftware.inl.gov/product/bison) - Flagship fuels performance code
+- Centipede - A tool to inform engineering scale simulations with atomistic data
+- [GRIFFIN](https://inlsoftware.inl.gov/product/griffin) - Reactor physics code co-developed between Argonne and Idaho National Laboratory
+- [GRIZZLY](https://inlsoftware.inl.gov/product/grizzly) - Nuclear Plant System Degradation Modeling
+- [MARMOT](https://inlsoftware.inl.gov/product/marmot) - Nuclear materials phase field and mechanics application
+- [PRONGHORN](https://inlsoftware.inl.gov/product/pronghorn) - Transient prismatic and pebble bed reactor analysis code
+- [RELAP7](https://inlsoftware.inl.gov/product/relap7) - Next Generation nuclear reactor system safety analysis code (compressible flow)
+- SAM - Advanced nuclear reactor system analysis code (weakly-compressible flow)
+- [Sockeye](https://inlsoftware.inl.gov/product/sockeye) - Multi-fidelity studies of heat pipes for nuclear applications
+- SWIFT - A thermo-chemistry tool for metal hydride moderators
```

Comment:
Not sure if you do this everywhere, but might want to add "developed at LANL" like you do for Gardensnake.

Reply:
Speaking of this, have you reached a deal on the naming clash? @tophmatthews @dschwen

Reply:
Tomorrow. Bring your popcorn.